server/server/src/main/kotlin/org/jetbrains/bsp/bazel/server/bsp/utils/BUILD

```
@@ -3,13 +3,16 @@ load("@rules_kotlin//kotlin:jvm.bzl", "kt_jvm_library")
 kt_jvm_library(
     name = "utils",
     srcs = [
+        "DelimitedMessageReader.kt",
         "InternalAspectsResolver.kt",
     ],
     visibility = ["//server/server:__subpackages__"],
     deps = [
         "//server/bazelrunner",
         "//server/commons",
         "//server/server/src/main/kotlin/org/jetbrains/bsp/bazel/server/bsp/info",
+        "@maven//:io_grpc_grpc_protobuf",
+        "@maven//:org_jetbrains_kotlinx_kotlinx_coroutines_core",
```

```suggestion
        "@server_maven//:org_jetbrains_kotlinx_kotlinx_coroutines_core",
```

server coroutine libraries are separate because we have problems in the client (because IJ SDK brings its own custom coroutine lib)